pagefaultgames/pokerogue
# [Balance] Update egg moves

## What are the changes?

The latest batch of egg move changes, courtesy of the Balance Team.

## Why am I doing these changes?



This is the usual balancing adjustments for egg moves, which are changed based on feedback.

## What did change?


Around 180 Pokémon had their egg moves revised. Check the changed file for all the specific changed moves.

### Screenshots/Videos


N/A

## How to test the changes?




Check the Pokémon on the starter select screen to see all the changed egg moves.

## Checklist
